.github/workflows/cargo-update.yml

```
@@ -22,8 +22,8 @@ jobs:
         continue-on-error: true
         run: |
           git config user.name 'Phylum Bot'
```

```suggestion
          git config user.name 'phylum-bot'
```
Not sure if this matters as well...just a suggestion.

---

Good catch. It matters if we want these to perfectly match the commits created by the GitHub web flow, so we might as well make the change. But I don't think it matters for user matching or display.